README.md

```diff
@@ -1,16 +1,13 @@
 [![Build Status](https://travis-ci.org/StartupWeekend15/Decision.svg?branch=master)](https://travis-ci.org/StartupWeekend15/Decision)
+### Choozeroo 
 
-### Decision App
-
-Don't be a headless chicken!
+Helping solve the issue of indecisiveness
 
 ### How to run
-
-* `npm install`
-* Copy .env file to the root directory of the project
-* Run
-  `npm start`
-  * For development: `npm run start-dev`
```

Comment:
I probably wouldn't remove these lines - it's important that we have directions on running the app locally